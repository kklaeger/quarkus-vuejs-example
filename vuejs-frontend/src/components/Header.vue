<template>
  <section class="section">
    <h1 class="title">User List</h1>
  </section>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
@Component({
  components: {}
})
export default class Header extends Vue {}
</script>

<style scoped lang="scss"></style>
