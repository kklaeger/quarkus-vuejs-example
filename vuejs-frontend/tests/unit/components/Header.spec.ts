import { shallowMount } from "@vue/test-utils";
import Header from "@/components/Header.vue";

describe(Header.name + " component", () => {
  it("should render successfully.", () => {
    const wrapper = shallowMount(Header);
    expect(wrapper).toMatchSnapshot();
  });
});
