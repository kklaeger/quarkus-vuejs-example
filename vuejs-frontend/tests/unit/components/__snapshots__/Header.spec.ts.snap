// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render successfully. 1`] = `
<section class="section">
  <h1 class="title">User List</h1>
</section>
`;
